schema/config-schema.json

```diff
@@ -50,13 +50,15 @@
         "process": {
             "type": "object",
             "required": [
-                "cwd",
-                "args"
+                "cwd"
```

Comment:
@crosbymichael - Is there a way to do `required: oneOf{ "args", "commandLine" }` or something here?